source/_patterns/02-molecules/navigation/footer-nav/footer-nav.twig

```diff
@@ -0,0 +1,9 @@
+<div class="jcc-footer__menu tablet:grid-col">
+  <ul class="jcc-footer__nav">
+    {% for item in footer_nav.items %}
+    <li class="jcc-footer_nav__item">
+      <a href="{{ item.url }}">{{ item.name }}</a>
+    </li>
+    {% endfor %}
+  </ul>
+</div>
```

Comment:
Just a head's up that this menu can't be consumed by Drupal. As we move toward Trial Courts we should consider developing a menu component that Drupal can use. See https://github.com/chapter-three/jcc-srl/blob/master/web/themes/custom/atrium/templates/navigation/menu--main.html.twig

Reply:
@zakiya as I mentioned to @slowbot that this footer is temporary for the Alphas & Beta until the footer design becomes available. We need to get two critical links out there since the new services are being viewed by the public.